pipeline/metadata/maxmind.py

```diff
@@ -27,7 +28,10 @@ def _maxmind_reader(filepath: str) -> geoip2.database.Reader:
 
   # MaxMind Reader will only take a filepath,
   # so we need to write the file to local disk
-  disk_filename = os.path.join('/tmp', os.path.basename(filepath))
+  # give files on disk randomly generated names so they don't get clobbered
+  random_string = uuid.uuid4().hex
+  disk_filename = os.path.join('/tmp',
```

Comment:
If you want to use a temporary file, use tempfile: https://docs.python.org/3/library/tempfile.html

Reply:
By the way, they provide a context manager that will delete the file afterwards, which will be useful here.

Reply:
switched to using tempfile